benches/spec_samples/README.md

```diff
@@ -0,0 +1,285 @@
+CommonMark
```

Comment:
Yes, more test files finally! Could you move these to `/third_party/CommonMark` so that it's clear where they came from?

edit: just realized that the javascript tests may have a different origin. If so, could you please add a separate third party folder for them, including a license file?

Reply:
The newly added samples are now correctly attributed to markdown-it, and a license has been added to reflect this.